release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -44,6 +44,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-10-azure-gen2-vm"]
+==== Support for Azure generation 2 VMs
```

Comment:
Should you spell-out `virtual machines (VM)` in the first reference? (Or is it too obvious what it means?)